instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy

```diff
@@ -73,6 +80,106 @@ abstract class AbstractAws2ClientTest extends InstrumentationSpecification {
 
   abstract void configureSdkClient(SdkClientBuilder builder)
 
+  def "send DynamoDB #operation request with builder #builder.class.getName() mocked response"() {
+    setup:
+    configureSdkClient(builder)
+    def client = builder
+      .endpointOverride(server.address)
+      .region(Region.AP_NORTHEAST_1)
+      .credentialsProvider(CREDENTIALS_PROVIDER)
+      .build()
+    responseBody.set("")
+    def response = call.call(client)
+
+    if (response instanceof Future) {
+      response = response.get()
+    }
+
+    expect:
+    response != null
+    response.class.simpleName.startsWith(operation)
+    assertDynamoDbRequest(service, operation, path, method, requestId)
+
+    where:
+    [service, operation, method, path, requestId, builder, call] << dynamoDbRequestDataTable()
+  }
+
+  def "send DynamoDB #operation async request with builder #builder.class.getName() mocked response"() {
+    setup:
+    configureSdkClient(builder)
+    def client = builder
+      .endpointOverride(server.address)
+      .region(Region.AP_NORTHEAST_1)
+      .credentialsProvider(CREDENTIALS_PROVIDER)
+      .build()
+    responseBody.set("")
+    def response = call.call(client)
+
+    if (response instanceof Future) {
+      response = response.get()
+    }
+
+    expect:
+    response != null
+    assertDynamoDbRequest(service, operation, path, method, requestId)
+
+    where:
+    [service, operation, method, path, requestId, builder, call] << dynamoDbRequestDataTable()
+  }
+
+  def assertDynamoDbRequest(service, operation, path, method, requestId) {
+    assertTraces(1) {
+      trace(0, 1) {
+        span(0) {
+          operationName "$service.$operation"
+          spanKind CLIENT
+          errored false
+          parent()
+          attributes {
+            "${SemanticAttributes.NET_PEER_NAME.key()}" "localhost"
+            "${SemanticAttributes.NET_PEER_PORT.key()}" server.address.port
+            "${SemanticAttributes.HTTP_URL.key()}" { it.startsWith("${server.address}${path}") }
```

Comment:
Can we set https://github.com/open-telemetry/opentelemetry-specification/blob/bc272ab52869df2145065d55300d721ff008e34b/specification/trace/semantic_conventions/span-general.md#nettransport-attribute ?

Reply:
Good question! What do you think about adding it to (generic HTTP tracer base class) - HttpClientTracer ? This way all HTTP based tracers would set "TCP.IP".

Reply:
That is a good idea

Reply:
Will do :)